e2e-tests/graphql-spring-examples2/pom.xml

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <!-- TODO: Parent should be changed TO evomaster, for some reasons it wont work right now !-->
```

Comment:
this should really be fixed. see pom.xml of spring-example as a reference. i can help you debugging the problem

Reply:
when I change the parent to EvoMaster I'm getting the following error, and I have NO idea how to fix that, I'll really appreciate your help:
https://pastebin.com/Dh4Gqpvp

Reply:
that would likely be dependent on a current issue in old libs.
adding the following would likely fix the issue:
```
 <dependency>
            <groupId>javax.validation</groupId>
            <artifactId>validation-api</artifactId>
            <version>2.0.1.Final</version>
        </dependency>
```

Reply:
Yes that fixed the problem. thanks

Reply:
It fixed the problem I had but now I can't run PetShopEMTest file and I get the following error:
https://pastebin.com/g6DPT0pw

I had this error before and I fixed it by excluding graphql-java package from importing from evomaster.core (which inside this package we need graphql-java for schema parsing) but now I have this error by running PetShopEMTest file. Do you have any idea for this issue ?

Reply:
hi,
 in those logs I am seeing:
```
Caused by: java.lang.ClassNotFoundException: kotlin.reflect.KotlinReflectionInternalError
```
do you have the following in the pom?
```
<dependency>
                <groupId>org.jetbrains.kotlin</groupId>
                <artifactId>kotlin-reflect</artifactId>
                <version>${kotlin.version}</version>
</dependency>
```

Reply:
hi,
Yes i have it in e2e-test pom file.

`        <dependency>
            <groupId>org.jetbrains.kotlin</groupId>
            <artifactId>kotlin-reflect</artifactId>
        </dependency>
`

Reply:
Hi,
 I checked out your branch, and tried to run `PetShopEMTest`. However, I get a very different error, which is about Swagger (which shouldn't be there for GraphQL) 

![image](https://user-images.githubusercontent.com/14858082/63588986-2bb57280-c5a8-11e9-9106-c5c9ef4974b1.png)

Reply:
Yes, I changed the default sampler to RestSampler in that commit but now I have added a new problem to the EvoMaster but I haven't pushed the new changes (since I still have this bug) if you can check the error personally I can push the new commit.

Reply:
hi. yes, please commit to the branch, even if it has errors (that's the point of using branches instead of pushing directly to master).

Reply:
hi, I have pushed the last changed.
The main problem that I am facing right now is that I can run PetShop main spring file (GraphqlSpringBootTutorial.java) but I can't run PetShopEMTest (since it can't run the spring project) .

Reply:
it looks like that the GraphQL libraries you use have a old version of Jackson, not compatible with current version of Kotlin.
Adding the following dependency should fix the issue:

```
       <dependency>
            <groupId>com.fasterxml.jackson.module</groupId>
            <artifactId>jackson-module-kotlin</artifactId>
            <version>${jackson.version}</version>
        </dependency>
```

Reply:
Hi,
thank you for your help that actually solved the problem.
Now I believe its ready now and we can continue.